handlers/discord/discord.go

```diff
@@ -0,0 +1,243 @@
+package discord
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"strings"
+
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+	"github.com/nyaruka/courier/utils"
+	"github.com/nyaruka/gocommon/urns"
+	"github.com/pkg/errors"
+)
+
+const (
+	contentURLEncoded = "urlencoded"
+	contentJSON       = "json"
+
+	configMOFromField = "mo_from_field"
+	configMOTextField = "mo_text_field"
+	configMODateField = "mo_date_field"
+
+	configMOResponseContentType = "mo_response_content_type"
+	configMOResponse            = "mo_response"
+)
+
+var defaultFromFields = []string{"from", "sender"}
```

Comment:
Same here, since you are controlling both sides may as well just be specific about it.

Reply:
I've made these changes. You're totally right that we don't need these to be configurable as we control both sides.